task/07-yield-tasks.js

```diff
@@ -33,10 +33,24 @@
  *
  */
 function* get99BottlesOfBeer() {
-    throw new Error('Not implemented');
+    let line = true;
+    for(let i = 99; i>=0; --i ) {
+        if (line) { 
+            i<99 ? i+=1 : i;
```

Comment:
Never do the assignments inside teranry operator. Ternary operator, result of ternary operator should always be assigned to some variable e.g. res = condtion ? res1 : res2;
![image](https://user-images.githubusercontent.com/7272208/88691143-242a1200-d105-11ea-8567-c12625e3e1f0.png)